src/utils/index.ts

```diff
@@ -4,7 +4,7 @@ export * from './viewportUtils';
 export * from './keyboardUtils';
 export * from './selectedCellUtils';
 
-export function assertIsValidKey<R>(key: unknown): asserts key is keyof R {
+export function assertIsValidKey<R, SR>(key: unknown): asserts key is keyof R {
```

Comment:
```suggestion
export function assertIsValidKey<R>(key: unknown): asserts key is keyof R {
```
The generic is unused here.